engine/src/main/java/org/hibernate/search/query/dsl/sort/SortNativeContext.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.query.dsl.sort;
+
+/**
+ * @author Emmanuel Bernard emmanuel@hibernate.org
+ */
+public interface SortNativeContext extends SortAdditionalSortFieldContext, SortTermination {
```

Comment:
Ok fair enough. Just trying to gauge were this is going, as I expected a much simpler Sort API.. but I guess you're right I hadn't though about this aspect.

Let's try to make concrete preparations for deprecating the traditional `createFullTextQuery` during the 6.0 works then.